src/main/java/org/systemexception/crudapplication/servlet/HelloWorld.java

```diff
@@ -16,7 +16,7 @@
  */
 public class HelloWorld extends HttpServlet {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Each class should declare at least one constructor](https://www.codacy.com/app/leo_4/crud-application/file/1805227035/issues/source?bid=3039358&fileBranchId=3073638#l17)